apps/charterafrica/src/components/Tool/OrgThemeAndOperatingCountries.js

```diff
@@ -23,14 +23,16 @@ const OrgThemeAndOperatingCountries = React.forwardRef(
             width: "auto",
           }}
         />
-        <RichTypography
+        <Link
+          href={organisation?.link?.href || "#"}
+          component={RichTypography}
```

Comment:
Not sure I follow here: `Link` using `RichTypography` as component? `Link` is already built on [top](https://mui.com/material-ui/react-link/#basic-links) of `Typography`, do we need both?

Reply:
Then we do not need RichTypography